modules/custom/openy_search/openy_google_search/openy_google_search.module

```diff
@@ -82,13 +85,21 @@ function openy_google_search_help($route_name, RouteMatchInterface $route_match)
  */
 function openy_google_search_openy_search_theme_configuration_alter(&$overrides) {
   $search_config = \Drupal::configFactory()->get('openy_google_search.settings');
-  $search_page_id = $search_config->get('search_page_id');
-  $langcode = Drupal::languageManager()->getCurrentLanguage()->getId();
-  $alias = ltrim(Drupal::service('path.alias_manager')->getAliasByPath('/node/' . $search_page_id, $langcode), '/');
+  $alias = ltrim(_get_alias(), '/');
 
   $overrides = [
     'search_query_key' => $search_config->get('search_query_key'),
     'search_page_alias' => $alias,
     'display_search_form' => 1,
   ];
 }
+
+/**
+ * Gets alias for results search page id saved in module settings config.
+ */
+function _get_alias() {
```

Comment:
Please use Drupal standards with function naming, the name should be started with the module name.

Reply:
Fixed